.github/workflows/release.yml

```diff
@@ -49,32 +52,18 @@ jobs:
           - chrome
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version-file: '.nvmrc'
-          cache: npm
-      - name: install
-        run: npm ci || npm install
-      - run: npm run build --if-present
-      - name: Update extension’s meta
-        run: >-
-          npx dot-json@1 $DIRECTORY/manifest.json version ${{
-          needs.Version.outputs.version }}
-      - name: Submit
-        run: |
-          case ${{ matrix.command }} in
-            chrome)
-              cd $DIRECTORY && npx chrome-webstore-upload-cli@2 upload --auto-publish
-              ;;
-            firefox)
-              cd $DIRECTORY && npx web-ext-submit@7
-              ;;
-          esac
+      - uses: actions/download-artifact@v3
+      - name: Submit to Mozilla
+        if: matrix.command == 'firefox'
+        run: cd artifacts/built && npx web-ext-submit@7
```

Comment:
The `built` artifact is created correctly in the previous job, but `artifacts/built` doesn't exist. I assume that's how a parameter-less `actions/download-artifact` would work.